src/corehost/common/pal.windows.cpp

```diff
@@ -62,12 +84,27 @@ bool pal::getcwd(pal::string_t* recv)
     return false;
 }
 
-bool pal::load_library(const char_t* path, dll_t* dll)
+bool pal::load_library(const string_t* in_path, dll_t* dll)
```

Comment:
i would like to keep the string_t type as it is more convenient to call methods on it